Engineering-Reference/M365DestinationAO.md

```diff
@@ -77,87 +77,85 @@ Default: Blank
 
 The access key for the Azure storage account.
 
-Change Conditions: Only avialable if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
+Change Conditions: Only available if the option SharePoint Migration API is enabled and SharePoint Storage is set to Azure Hosted. 
 
 ### Retry Backoff (seconds) <a name="sretryback"></a>
 {: .no_toc }
 Default Value: 3
 
 SharePoint Migration API Retry Backoff in seconds.
 
-Change Conditions: ??
+Change Conditions: Retry backoff time can be changed. Can cause performance issues, not recommended.
```

Comment:
Retry backoff time can be changed. Can cause performance issues, not recommended to change unless instructed by support.

Reply:
done